apps/ejabberd/src/node_push.erl

```diff
@@ -207,20 +208,16 @@ is_allowed_to_publish(PublishModel, Affiliation) ->
 parse_form(undefined) ->
     #{};
 parse_form(Form) ->
-    IsForm = ?NS_XDATA == exml_query:attr(Form, <<"xmlns">>),
-    IsSubmit = <<"submit">> == exml_query:attr(Form, <<"type">>),
```

Comment:
I was referring to [XEP-0357](https://xmpp.org/extensions/xep-0357.html#publishing). Multiple implementations could already exist that do not fit this requirement.

[Please be liberal in what you accept from others.](https://en.wikipedia.org/wiki/Robustness_principle)